### PR TITLE
Add submenu collapsed behavior

### PR DESCRIPTION
**Pull request by sardell**
## Contributor Comments
Currently, active items in the central navigation have styling applied to them (eg. green arrow on the left side, lighter background color, etc). However, there are not styles to handle active submenu items (ie. collapse menu items) in the central navigation.

Per our internal discussion, I implemented the following behavior:
* When the navigation is expanded, an active submenu item has the same styling as an active nav item.
* When the navigation is collapsed, the parent of the active submenu item reveals a green active arrow.

## Testing
1. Clone this branch locally.
1. Start up the project and open localhost:8000
1. In the index.html file (or in your inspector) add `active` as a class to the .navbar-parent and a .nav-sublink (see screenshot below).
Screen Shot 2019-03-19 at 12 35 38 PM
1. You should see the following behavior when opening/closing the central navigation.
submenu

## Further Considerations
I did not implement "sticky" behavior so that submenus remain open for a few seconds after hover. I avoided doing this because of time constraints. We should open an issue and address this in the future when we have time.

The application of the `active` class should be handled in Javascript. However, I think that should be implemented in the application consuming this library, as most modern JS frameworks have routers to handle active states.

IMHO, when the menu collapses, it's sort of jarring right now. If others agree, we should work to smooth that out a little with property transitions, if possible.

### CODE REVIEW

**Review by jgolieb — CHANGES_REQUESTED** [review]
This change is based on introducing new classes, which I'm not really a fan of unless absolutely necessary. Also, your CSS isn't Sass-y. If you don't have the time to address these issues, I can work on them. (I am doing other work on the navbar because of Fluid 2.0, so I don't really want to merge this until that is done anyway.)

**Comment by sardell** [comment]
@jgolieb Good point about avoiding new classes. I neglected to consider the fact that we refer to the Bootstrap docs for html examples, so the chances are high that a user wouldn't know of the new class existing. 

Leaving the styles flat was sort of intentional on my part, and I figured it would come up in the review. I can easily create a new shallow nest at the end of the file. If you would prefer them to be included in the existing nest, I can do that too, but I wanted to avoid doing that while initially writing out the styles. As a new contributer, blocks nested that deeply can be hard to parse and remember right off the bat. That isn't intended as a criticism to your approach, as I know you're trying to work with existing classes and html structures from the Bootstrap documentation with as little change as possible. It's just hard to quickly customize and/or extend for project newbies like me.

All that said, if you're already working with the navbar and have the capacity to take on these issues, I would greatly appreciate it. If not, I can get to it within the next few days.

**Comment by jgolieb** [comment]
@sardell I recognize the difficulty of working with the nesting structure for the navbar. That thing is a beast and easily the most complicated thing in the project. I have a couple other things to finish up before I would get to this, so whichever one of us gets to it first, we'll just let the other know.

**Review by mihai-mihalache — DISMISSED** [review]
